composer.json

```diff
@@ -8,22 +8,21 @@
     "require": {
         "php": ">=7.4",
         "drupal/address": "^1.10",
-        "drupal/core": "^9.2",
+        "drupal/core": "^9.3",
         "openeuropa/oe_media": "^1.14"
```

Comment:
oe_content requires this at ^1.15, which bumps file_link to 2.0.4, which means we can get rid of line 66